### DISCUSSION TITLE
Using Expo Go on canary SDK 52

### DISCUSSION THREAD

**Discussion by anicioalexandre** [expo/expo#31778] [Archived: Questions] [answered]
Hello!

I am currently using a canary version of Expo (SDK 52) for testing purposes, but I'm encountering the following issue:

`Project is incompatible with this version of Expo Go.`

I understand that Expo Go may not fully support canary versions, but is there any way to get Expo Go to work with SDK 52? Are there any workarounds, configurations, or a specific Expo Go version that I could use to make this compatible?

**Comment by tsapeta** [comment, marked as answer]
Hey! No, we don't release canary versions of Expo Go. You can use them only with the development builds or bare RN apps.
I recommend to migrate from Expo Go to development builds anyway – Expo Go is good only to easily get started, for educational purposes, prototyping, hackathons etc.
